server/controllers/companyController.js

```diff
@@ -57,6 +57,23 @@ const getCompany = async (req, res) => {
   }
 };
 
+const getCompanies = async (req, res) => {
+  const { query } = req.params;
+
+  // Create a regular expression to find documents that start with the query
+  const regex = new RegExp("^" + query, "i"); // "i" for case-insensitive search
+
+  // Find companies that match the regex
+  const companies = await Company.find({ name: regex });
+
+  res.status(200).json(companies);
+
+  try {
```

Comment:
why an empty try catch..?

Reply:
I was supposed to put the code in there lmao my bad